pkg/cluster/connect.go

```diff
@@ -218,6 +218,8 @@ func (c *KubernetesClusterAPIImpl) createServiceAccountSecretIfNeeded(namespace
 		localize.NewEntry("ClientID", serviceAcct.GetClientId()),
 	))
 
+	currentService.PrintAccessCommands(serviceAcct.GetClientId())
```

Comment:
My own feeling is that this does not belong in this package and should be printed from the connect command. This package is a Kubernetes API client and by print stuff to the console it is mixing responsibility layers.

Reply:
I haven't seen requirement to have this as kube client. I know we have this interface for it but IMHO this is slightly over-designing something that is very tightly related to CLI command. There are many places in services where we do the same, use context etc. 
Sole purpose of this package is to serve as execution layer for commands.

Reply:
The thing that highlighted this for me is that we are passing down `currentService` as a new function argument for the sole purpose of printing the access commands. It does not relate to the core purpose of the function (creating a service when needed) and goes against the single-responsibility principle.

Reply:
Funny. Rama wanted it this way and I have recomended to do it in function also quoting single responsibility rule (dealing with service accounts)

#opinions :)

Reply:
@rkpattnaik780 to make final call on this

Reply:
Yeah, same result at the end of the day :)

Reply:
I agree with the method part, have updated the method to return a client ID if service account is created and print from connect command.